### PR TITLE
Trame Content

### PR DESCRIPTION
**Pull request by banesullivan**

### CODE REVIEW

**Comment by tkoyama010** [comment]
Thanks @banesullivan for your work. Please wait while I do a review.

File: tutorial/10_trame/README.rst [hunk 1]
```diff
@@ -3,11 +3,49 @@
...
 =================
 
-Trame is an open-source platform for creating interactive and powerful visual analytics applications. Based on Python, and leveraging platforms such as VTK, ParaView, and Vega, it is possible to create web-based applications in minutes.
+`Kitware's Trame <https://kitware.github.io/trame/index.html>`_ is an open-source platform for creating interactive and powerful visual analytics applications. Based on Python, and leveraging platforms such as VTK, ParaView, and Vega, it is possible to create web-based applications in minutes.
 
-.. tip::
+.. raw:: html
 
-    Learn more about `Kitware's Trame <https://kitware.github.io/trame/index.html>`_.
+    <iframe src="https://player.vimeo.com/video/764741737?muted=1" width="640" height="360" frameborder="0" allow="autoplay; fullscreen" allowfullscreen></iframe>
```
Comment (tkoyama010) [inline tutorial/10_trame/README.rst:10]: Cool video!

File: tutorial/10_trame/README.rst [hunk 2]
```diff
@@ -17,11 +55,57 @@ visualization in reactive web applications.
...
+          :alt: A simple Trame application with PyVista
+          :align: center
+
+
 .. note::
 
-    PyVista's Jupyter backend is powered by Trame! Our Jupyter backend is a micro Trame application.
+    PyVista's Jupyter backend is powered by Trame! If you've been using
+    PyVista in Jupyter lately, you've been using Trame all along -- our
+    Jupyter backend is a micro Trame application
```
Comment (akaszynski) [inline tutorial/10_trame/README.rst:105]:
```suggestion
    Jupyter backend is a micro Trame application.
```

**Comment by akaszynski** [comment]
Change the directory name due to https://github.com/pyvista/pyvista-tutorial/pull/128.

File: tutorial/10_trame/README.rst [hunk 3]
```diff
@@ -17,11 +55,57 @@ visualization in reactive web applications.
...
+        mesh = pv.Wavelet()
+
+        pl = pv.Plotter()
+        pl.add_mesh(mesh)
+
+        with SinglePageLayout(server) as layout:
+            with layout.content:
+                # Use PyVista's Trame UI helper method
+                #  this will add UI controls
+                view = plotter_ui(pl)
```
Comment (tkoyama010) [inline tutorial/10_trame/README.rst:90]: @banesullivan  For the Trame beginner, the other parts are intuitive, but using with is a little more thoughtful. I would like to see a detailed explanation of why. Is it for memory resource management?

File: .github/workflows/doc.yml [hunk 4]
```diff
@@ -73,6 +73,8 @@ jobs:
         run: |
           mkdir _notebooks
           find doc/source/tutorial -type f -name '*.ipynb' | cpio -p -d -v _notebooks/
+          find doc/source/tutorial/10_trame -type f -name '*.py' | cpio -p -d -v _notebooks/
```
Comment (tkoyama010) [inline .github/workflows/doc.yml:76]: I guess...
```suggestion
          find doc/source/tutorial/10_trame -type f -name '*.py' | cpio -p -d -v _notebooks/tutorial
```

**Pull request merged**